.github/workflows/code_quality.yml

```diff
@@ -0,0 +1,40 @@
+name: code quality
+on:
+    push:
+        branches:
+            - master
+            - staging
+            - trying
```

Comment:
Do we really need these checks for these branches? I can't come up with when it can be useful